src/toolkits/object_detection/object_detector.cpp

```diff
@@ -898,6 +897,10 @@ std::unique_ptr<data_iterator> object_detector::create_iterator(
 
 std::unique_ptr<compute_context> object_detector::create_compute_context() const
 {
+  //add this line to prevent evaluation for python model from crash
+  if (state.find("use_tensorflow") == state.end()){
+    return compute_context::create();
```

Comment:
Could you consult with Fareeha and Shreya? I don't think we want this logic here. Probably we can just remove "use_tensorflow" everywhere in the C++ code and just use `compute_context::create`